src/docs/sdk/performance/span-operations.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Span Operations"
+---
+
+Span operations are a short code identifying the type of operation the span is measuring. Span operations are low cardinality attributes - they should be as general as possible while still being human readable and useful.
+
+Operations are expected to follow [OpenTelemetry's semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/README.md) as much as possible.
+
+Below is a table containing a list of span operations that are recommended to be used.
+
+Operations are expected to generally take the form: `CATEGORY.ACTION`, where the categories and actions can have optional identifiers. For example, a redis database operation takes the form `db.sql.query`, where `db` is the category, `sql` is the category identifier, and `query` is the action. Another example is a mobile app rendering it's screen. It would have the operation `ui.load.hot`, where `ui` is the category, `load` is the action, and `hot` is the action identifier.
+
+It's important to keep categories consistent between sdks and integrations as they are used by Sentry in the [Operations Breakdown](https://docs.sentry.io/product/sentry-basics/tracing/event-detail/#operations-breakdown) feature. For example, both `db.init` and `db.query` will be categorized as database operations (`db`). The default operations breakdown config can be seen [here](https://github.com/getsentry/sentry/blob/809b7fe54c6f06cc1e4c503cf83ded896472a011/src/sentry/projectoptions/defaults.py#L74).
+
+Due to use of categories in the Sentry product, span categories are expected to be low cardinality data. **Categories should not be auto-generated**. SDK developers should feel free to automatically generate the actions and identifiers for span operations.
```

Comment:
Needs verification

- [ ] We start out saying "Span operations are low cardinality attributes", and this sentence here relaxes that expectation, essentially accepting high-cardinality span operations, which sounds contradictory at a minimum, and might have negative performance consequences.

Reply:
The important point here isn't that it's generated, but that the span categories stay low cardinality. I think just recommending that if a span category **is** generated, that it should only use low cardinality inputs (a static array like how we create the ops breakdown measurements) and not something like an `id`, `url`, etc. 

Since `generated` is a fairly conflated term, I'd perhaps rewrite this to state that:
- It should be low cardinality
- If you **are** generating this somehow, take care to not use high cardinality inputs